frontend/src/app/modules/notification/notification.module.ts

```diff
@@ -1,49 +1,10 @@
 import { NgModule } from '@angular/core';
 import { NotifierOptions, NotifierModule } from 'angular-notifier';
-const customNotifierOptions: NotifierOptions = {
- position: {
-  horizontal: {
-    position: 'right',
-    distance: 12
-  },
-  vertical: {
-    position: 'bottom',
-    distance: 12,
-    gap: 10
-  }
-},
-theme: 'material',
-behaviour: {
-  autoHide: 5000,
-  onClick: 'hide',
-  onMouseover: 'pauseAutoHide',
-  showDismissButton: true,
-  stacking: 4
-},
-animations: {
-  enabled: true,
-  show: {
-    preset: 'slide',
-    speed: 300,
-    easing: 'ease'
-  },
-  hide: {
-    preset: 'fade',
-    speed: 300,
-    easing: 'ease',
-    offset: 50
-  },
-  shift: {
-    speed: 300,
-    easing: 'ease'
-  },
-  overlap: 150
-}
-};
+import { customNotifierOptions } from '../../shared/constants';
 @NgModule({
   imports: [
-    NotifierModule.withConfig(customNotifierOptions)
+    NotifierModule.withConfig(customNotifierOptions as NotifierOptions)
```

Comment:
don't think it is good decision to force variable to some type
https://palantir.github.io/tslint/rules/no-object-literal-type-assertion/